### DISCUSSION TITLE
如何修改页面自适应宽度呢？

### DISCUSSION THREAD

**Discussion by gukaifeng** [ppoffice/hexo-theme-icarus#973] [Q&A]
在其他几个 issues 和 disscussions 中找到了修改文章部分、widget 宽度，但是这个修改后只在电脑上有效（标准尺寸），当宽度缩小，在到达手机尺寸之前，这个宽度的自适应还是和原来一样的，就会很突兀，有什么解决方案吗？
示例 gukaifeng.cn。

**Comment by ppoffice** [comment]
@gukaifeng 你可以尝试增大`$tablet`的值：
https://github.com/ppoffice/hexo-theme-icarus/blob/4.4.0/include/style/base.styl#L27

**Reply by gukaifeng** [reply]
> @gukaifeng 你可以尝试增大`$tablet`的值： https://github.com/ppoffice/hexo-theme-icarus/blob/4.4.0/include/style/base.styl#L27

不太行，无论我怎么尝试，都无法获得一个在多个尺寸下都合适的效果，主要问题还是在一个类似 ipad 宽度的时候左边组件宽度突然增大很多，限制最大宽度又会导致组件和文章部分间隔太大。
